source/guides/api-guidance/index.html.md.erb

```diff
@@ -109,6 +109,15 @@ An OpenAPI definition can then be used by documentation generation tools to disp
 
 Make sure that your API is secure. GDS provides specific guidelines in [API technical and data standards](https://www.gov.uk/guidance/gds-api-technical-and-data-standards#when-to-authenticate-your-api). As well as this general guide, we have provided specific guidance here on securing your DfE API in the [DfE Developer Hub documentation](https://dfe-developerhub.education.gov.uk/documentation/authorisation)
 
+##JSON Security
+
+GDS API standards recommend that [JSON is used](https://www.gov.uk/guidance/gds-api-technical-and-data-standards#use-json), APIs need to be designed so that JSON Objects are submitted rather than individual elements or arrays, JSON data needs to be parsed securely
```

Comment:
This sentence is quite long. Maybe it could be recast as a list, like
GDS API standards recommend that:

* JSON is used
* APIs are designed so that...
* JSON data is parsed securely...